ting_marc.module

```diff
@@ -346,3 +354,31 @@ function ting_marc_field_js() {
 
   drupal_json_output($result);
 }
+
+/**
+ * AJAX handler for author field load.
+ *
+ * @return array
+ *   Set of result authors by entity id.
+ */
+function ting_marc_authors_js() {
+  $result = array();
+
+  if (!empty($_POST['marc_authors'])) {
+    $ids = $_POST['marc_authors'];
+
+    foreach ($ids as $id) {
+      $object = ting_marc_load_object($id);
+
+      if (!empty($object)) {
+        $a = $object->getValue('100', 'a');
+        $h = $object->getValue('100', 'h');
+        if (!empty($a) && !empty($h)) {
+          $result[$id] = "$a $h";
```

Comment:
@NicolaeGrigoriu This is inverted author names, so you need to have ", " in between the fields as stated in the case: "[100.a], [100.h]".